lib/internal/test_runner/test.js

```diff
@@ -691,4 +691,6 @@ class Suite extends Test {
   }
 }
 
-module.exports = { kCancelledByParent, kDefaultIndent, kSubtestsFailed, kTestCodeFailure, Test, Suite, ItTest };
+module.exports = {
+  kCancelledByParent, kDefaultIndent, kSubtestsFailed, kTestCodeFailure, kHookFailure, Test, Suite, ItTest,
```

Comment:
Can we make this multiline to minimize the diff / risk of git conflicts for future PRs?